modules/web/src/app/shared/components/terminal/terminal-status-bar/template.html

```diff
@@ -22,29 +22,40 @@
       <i class="km-icon-warning"></i>
     </div>
     <div>
-      <p>The connection to your Web Terminal was lost.</p>
+
+      <ng-container *ngIf="isConnectionLost; else sessionExpiration">
+        <p>The connection to your Web Terminal was lost.</p>
+      </ng-container>
+
+      <ng-template #sessionExpiration>
+        <p>The web session will expire in {{DefaultSessionExpiration}} minutes</p>
```

Comment:
I'm not sure if we want to be that precise here. This value may change on backend side in the future and it's correct only at the begging.
```suggestion
        <p>The web session will expire in soon</p>
```

Reply:
i agree with you we don't have any to get default settings from backend so instead of adding explicit hardcoded value make no sense as it will become outdated and useless once changed from backed.